### PR TITLE
Deployment Launcher docs

### PR DESCRIPTION
**Pull request by paulbalaji**
#### Description

deployment launcher docs - DM me for preview link

#### Tests

improbadoc serve

#### Documentation

this is the docs

#### Primary reviewers
If your change will take a long time to review, you can name at most two primary reviewers who are ultimately responsible for reviewing this request. @ mention them.

### CODE REVIEW

File: docs/projects/fps/get-started/upload-launch.md [hunk 1]
```diff
@@ -52,74 +51,63 @@ Your `spatialos.json` should look something like this:
...
 > **TIP**: If you want to overwrite an existing assembly, ensure the `Force Upload` option is enabled.
 
-<p/>
+</p>
 
 > **TIP:** Depending on your network speed, the upload may take a little while (1-10 minutes) to complete.
 
-<%(Callout message="It’s finished uploading when You see an upload report printed in your Unity Console, for example:<br/><br/>**Uploaded assembly my_assembly to project unity_gdk successfully.**")%>
+<%(Callout message="
+It’s finished uploading when You see an upload report printed in your Unity Console, for example:
```
Comment (jessicafalk) [inline docs/projects/fps/get-started/upload-launch.md]:
```suggestion
It’s finished uploading when you see an upload report printed in your Unity Console, for example:
```

File: docs/modules/deployment-launcher/manage-deployments.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+### Stopping deployments
+
+When you press the `Stop deployment` button, the Deployment Launcher sends a request to stop the deployment chosen from the adjacent drop-down option.
+
+<img src="{{assetRoot}}assets/modules/deployment-launcher/stop-deployments-choice.png" style="margin: 0 auto; width: auto; display: block;" />
+
+<%(Callout type="warn" message="
+
+The list of deployments and corresponding metadata are **not** automatically refreshed. Ensure that you have refreshed the list manually before performing any action.
```
[the remark this answers is not in the document]
Comment (paulbalaji) [reply docs/modules/deployment-launcher/manage-deployments.md]: If someone stops it from the Console it doesn't magically update the UI - wanted to make people aware of that

File: docs/modules/deployment-launcher/launch-deployments.md [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+A simulated player deployment configuration exposes six additional fields:
+
+| Setting | Required? | Description |
+| --- | --- | --- |
+| Snapshot Path | ❌ | Path to the snapshot you wish to start the deployment with, relative from the root of your SpatialOS project. |
+| Launch Config | ✔️ | Path to the launch configuration you wish to start the deployment with, relative from the root of your SpatialOS project. |
+| Region | ✔️ | The geographical location in which your deployment launches. |
+| Tags | ❌ | Metadata that can be added to a deployment. Some tags have built-in functionality. For example, the `dev_login` tag is used to connect into deployment through the [development authentication flow](https://docs.improbable.io/reference/latest/shared/auth/development-authentication). |
+| Flag Prefix | ✔️ |  |
+| Worker Type | ✔️ |  |
```
[the remark this answers is not in the document]
Comment (paulbalaji) [reply docs/modules/deployment-launcher/launch-deployments.md]: we probably should have a guide on doing your own simplayers - when we do that it'd make sense to make the simplayer coordinator more generic to let users hook in

File: docs/modules/deployment-launcher/upload-assemblies.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+| Force Upload | Denotes whether to force upload this assembly.<br/><br/>If this is checked, an assembly that previously was uploaded with the same assembly name will be overwritten. |
+
+<%(/Expandable)%>
+
+<%(#Expandable title="Buttons")%>
+
+| Button name | Description |
+| --- | --- |
+| Generate | When pressed, this generates an **Assembly Name** for you based on your SpatialOS project name and a timestamp. |
+| Assign assembly name to deployments | When pressed, this copies the current value of **Assembly Name** to _all_ deployments you currently have [configured in the next section]({{urlRoot}}/modules/deployment-launcher/launch-deployments). |
```
Comment (jessicafalk) [inline docs/modules/deployment-launcher/upload-assemblies.md]:
```suggestion
| Assign assembly name to deployments | When pressed, this copies the current value of **Assembly Name** to _all_ deployments you currently have configured as described in [the next section]({{urlRoot}}/modules/deployment-launcher/launch-deployments). |
```

or something similar, just find the usage of `currently` and `next` right next to each other slightly confusing

File: docs/modules/deployment-launcher/launch-deployments.md [hunk 5]
```diff
@@ -0,0 +1,208 @@
...
+
+<img src="{{assetRoot}}assets/modules/deployment-launcher/dpl-configs.png" style="margin: 0 auto; width: auto; display: block;" />
+
+<%(#Expandable title="Fields")%>
+
+| Field | Required? | Description |
+| --- | --- | --- |
+| Assembly Name | ✔️ | The identifier of an assembly you have uploaded. |
+| Deployment Name | ✔️ | The name you wish to give to the deployment. |
+| Snapshot Path | ❌ | Path to the snapshot you wish to start the deployment with, relative from the root of your SpatialOS project. |
```
Comment (jessicafalk) [inline docs/modules/deployment-launcher/launch-deployments.md]: add a link to some glossary or similar for snapshot

File: docs/modules/deployment-launcher/launch-deployments.md [hunk 6]
```diff
@@ -0,0 +1,208 @@
...
+
+At the bottom of the deployment configurations panel, there is a **Launch deployment** button. You can select which which deployment configuration you wish to launch and then select **Launch deployment**.
+
+<img src="{{assetRoot}}assets/modules/deployment-launcher/choose-launch-config.png" style="margin: 0 auto; width: auto; display: block;" />
+
+## Expected behaviour
+
+### Input validation
+
+The deployment launcher validates your deployment configurations as you edit them in the Unity Editor. If any field fails validation, errors similar to those highlighted below are shown and the **Launch deployment** button is disabled.
```
Comment (jessicafalk) [inline docs/modules/deployment-launcher/launch-deployments.md]:
```suggestion
The deployment launcher validates your deployment configurations as you edit them in the Unity Editor. If any field fails validation, errors similar to those highlighted below are shown and the **Launch deployment** button is disabled until you resolved all errors.
```

File: docs/modules/deployment-launcher/launch-deployments.md [hunk 7]
```diff
@@ -0,0 +1,208 @@
...
+Assembly reloading locked.
+```
+
+The standard output and standard error from the command is forwarded to the Unity Console.
+
+<%(Callout message="
+When a deployment successfully launches:
+
+* The [Console](https://docs.improbable.io/reference/latest/shared/glossary#console) page for the deployment automatically opens in your browser.
+* The notification at the bottom of the Deployment Launcher window will disappear.
```
Comment (jessicafalk) [inline docs/modules/deployment-launcher/launch-deployments.md]:
```suggestion
* The notification at the bottom of the Deployment Launcher window disappears.
```

File: docs/modules/deployment-launcher/manage-deployments.md [hunk 8]
```diff
@@ -0,0 +1,98 @@
...
+
+# Manage deployments
+
+Open the Deployment Launcher window by selecting **SpatialOS** > **Deployment Launcher** from the Unity Editor menu.
+
+## UI walkthrough
+
+The section of the Deployment Launcher window that you use to manage live deployments can be found under the **Live Deployments** label.
+
+Initially, the Deployment Launcher does not know of any deployments. It will only update its known list of deployments when requested.
```
Comment (jessicafalk) [inline docs/modules/deployment-launcher/manage-deployments.md]:
```suggestion
Initially, the Deployment Launcher does not know of any deployments. It only updates its known list of deployments when requested.
```

File: docs/modules/deployment-launcher/manage-deployments.md [hunk 9]
```diff
@@ -0,0 +1,98 @@
...
+
+When there are live cloud deployments running, this section is updated with a list of these live deployments and some metadata about them. The deployment metadata is **read-only**.
+
+<img src="{{assetRoot}}assets/modules/deployment-launcher/manage-deployments.png" style="margin: 0 auto; width: auto; display: block;" />
+
+<%(#Expandable title="Metadata")%>
+
+| Metadata | Example |
+| --- | --- |
+| Deployment name | The name of the given cloud deployment. For example, `testmobile13052019`. |
```
Comment (jessicafalk) [inline docs/modules/deployment-launcher/manage-deployments.md]: just leaving a comment here that I still don't like `testmobile13052019` as an example xD

**Pull request merged**